optaplanner-docs/src/main/asciidoc/Integration/Integration-chapter.adoc

```diff
@@ -508,6 +508,10 @@ optaplanner.solver-config-xml::
 A classpath resource to read the solver configuration XML.
 Defaults to `solverConfig.xml`.
 If this property isn't specified, that file is optional.
+optaplanner.score-drl::
+A classpath resource to read the score DRL.
+Defaults to `constraints.drl`.
+Do not define this property when a `ConstraintProvider`, `EasyScoreCalculator` or `IncrementalScoreCalculator` class exists.
```

Comment:
Great work. Mind if I add this issue on your backlog (not for this PR)? https://issues.redhat.com/browse/PLANNER-2288

Reply:
No problem, in fact I have already created one (PLANNER-2284) - see one of my earlier responses on this PR.